how to define functions

```diff
@@ -0,0 +1,62 @@
+vals = (3, 7)
+
+f x = (x, x + 1)
+
+g :: Int -> (Int, Int)
+g x = (x+1, x)
```

Comment:
Type signatures: Yes, I think we should talk about them on this page. I like how you've included a type signature for `g` but not for `f` so we can highlight how type signatures are optional.

There's one complication I'd like to address: the type of `g` with the type signature (`Int -> (Int, Int)`) is different from the type that would be inferred implicitly (`Num a => a -> (a, a)`). This is a more subtle concern than I'd like to impose on readers at this point, so I'm wondering if we could instead do this demonstration of optional type signatures on an example where the inferred type would be monomorphic.